frontend/src/store/index.js

```
@@ -986,18 +1012,30 @@ const actions = {
       })
   },
   subscribeShoots ({ dispatch, commit, state }) {
-    return EmitterWrapper.shootsEmitter.subscribeShoots({ namespace: state.namespace, filter: getFilterValue(state) })
+    return new Promise((resolve, reject) => {
```

async/await + ignore error in catch

---

✔️